proposals/ecosystem/index.bs

```diff
@@ -1805,7 +1805,7 @@ can be invoked with the following properties:
   <pre highlight="turtle">
   <#notebook-access-decorator-index>
       a eco:AccessDecoratorIndex ;
-      eco:defaultLanguage "en" ;
+      eco:defaultLanguage "en"^^xsd:lang ;
```

Comment:
`rdf:langString`

Reply:
I'm not sure what's intended actually. xsd:lang doesn't exist but there is xml:lang. Is it supposed to be rdf:langRange instead of eco:defaultLanguage?

Reply:
I think this was meant to be xsd:language. They both ([`xml:lang`](https://www.w3.org/TR/2008/REC-xml-20081126/#sec-lang-tag), [`xsd:langauge`](https://www.w3.org/TR/xmlschema-2/#language)) expect BCP47 language codes (noting that [`BCP47` obsoletes `RFC3066`](https://www.w3.org/International/articles/language-tags/)) but `xsd` is used all time in RDF.

Reply:
Updated to `xsd:language`